libs/api_common/api_common.cpp

```diff
@@ -204,3 +204,15 @@ ebpf_verify_program(
         return false;
     }
 }
+
+prevail::ebpf_verifier_options_t
+ebpf_get_default_verifier_options(ebpf_verification_verbosity_t verbosity)
+{
+    prevail::ebpf_verifier_options_t verifier_options = {0};
+    verifier_options.assume_assertions = verbosity < EBPF_VERIFICATION_VERBOSITY_VERBOSE;
```

Comment:
Is it possible to add a comment why `assume_assertions` is linked to verbosity?

Reply:
Assume assertions gets rid of a lot of the false positive warnings (i.e. things that failed because a previous assertion failed).